app.js

```diff
@@ -0,0 +1 @@
+console.log('Hello world')
```

Comment:
User the logger from the nodejs module rather than console.log

Reply:
Nodejs modules are terrible. Console is better.